src/data/polynomial/monic.lean

```diff
@@ -131,6 +135,84 @@ by rw [eq_C_of_degree_le_zero this, ← nat_degree_eq_zero_iff_degree_le_zero.2
 
 end comm_semiring
 
+section comm_ring
+variables [comm_ring R]
+namespace monic
+
+lemma coeff_nat_degree {p : polynomial R} (hp : p.monic) : p.coeff (p.nat_degree) = 1 := hp
+
+@[simp]
+lemma degree_one {p : polynomial R} (hp : p.monic) :
+p.nat_degree = 0 ↔ p = 1 :=
+begin
+  split; intro h,
+  swap, { rw h, exact nat_degree_one },
+  have : p = C (p.coeff 0),
+  { rw ← polynomial.degree_le_zero_iff,
+    rwa polynomial.nat_degree_eq_zero_iff_degree_le_zero at h },
+  rw this, convert C_1, rw ← h, apply hp,
+end
+
+lemma nat_degree_mul_eq [nontrivial R] {p q : polynomial R} (hp : p.monic) (hq : q.monic) :
+(p * q).nat_degree = p.nat_degree + q.nat_degree :=
+by { apply nat_degree_mul_eq', rw [hp.leading_coeff, hq.leading_coeff], simp }
+
+lemma next_coeff_mul {p q : polynomial R} (hp : monic p) (hq : monic q) :
+next_coeff (p * q) = next_coeff p + next_coeff q :=
+begin
```

Comment:
Notice the inconsistent naming here. Surely neither or both should have the `_eq` suffix! I think neither is the standard.